.ci/test.yml

```
@@ -90,6 +90,25 @@ jobs:
     displayName: 'Set UsingAzAuth environment variable'
     condition: eq(${{ parameters.useAzAuth }}, true)
 
+  - task: AzurePowerShell@5
+    inputs:
+      azureSubscription: PSResourceGetACR
+      azurePowerShellVersion: LatestVersion
+      ScriptType: InlineScript
+      inline: |
+        $acrRepositoryNamesFolder = Join-Path -Path ([Environment]::GetFolderPath([System.Environment+SpecialFolder]::LocalApplicationData)) -ChildPath 'TempModules'
+        if (Test-Path -Path $acrRepositoryNamesFolder) {
+          Write-Verbose -Verbose "Deleting existing temp acr repository names folder path: $acrRepositoryNamesFolder"
+          Remove-Item -Path $acrRepositoryNamesFolder -Recurse -Force -ErrorAction Ignore
+        }
+        if (! (Test-Path -Path $acrRepositoryNamesFolder)) {
+          Write-Verbose -Verbose "Creating new folder for acr repository names file to be placed with path: $acrRepositoryNamesFolder"
+          $null = New-Item -Path $acrRepositoryNamesFolder -ItemType Directory
+        }
```

```suggestion
        Write-Verbose -Verbose "Creating new folder for acr repository names file to be placed with path: $acrRepositoryNamesFolder"
        $null = New-Item -Path $acrRepositoryNamesFolder -ItemType Directory
```

It seems that at this point, the folder will never be there. So no need to check.